js/__internal/grids/data_grid/export/m_export.ts

```diff
@@ -30,7 +30,14 @@ export class DataProvider {
 
   private readonly _selectedRowsOnly: any;
 
-  private _options: any;
+  private _options!: {
+    isHeadersVisible: any;
+    rtlEnabled: any;
+    summaryTexts: any;
+    groupColumns: any;
+    items: any;
+    columns: any;
+  };
```

Comment:
Let's add obvious types right away, e.g:
```typescript
  private _options!: {
    isHeadersVisible: boolean;
    rtlEnabled: boolean;
    summaryTexts: any;
    groupColumns: any;
    items: any;
    columns: any;
  };
```

And question.
Can we use the ```unknown``` instead of the ```any``` here?
Will it break something?